app/src/main/java/org/mozilla/focus/settings/SettingsFragment.java

```diff
@@ -185,7 +191,7 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
 
             // The easiest way to ensure we update the language is by replacing the entire fragment:
             getFragmentManager().beginTransaction()
-                    .replace(R.id.container, new SettingsFragment())
+                    .replace(R.id.container, SettingsFragment.newInstance(null, R.xml.settings, R.string.menu_settings))
```

Comment:
nit:  We use duplicate these resource arguments twice (putting them in danger of getting out of sync), here and in `SettingsActivity.onCreate`, when all we're really trying to say is, "Show top level settings".

It may be outside the scope of this quick fix, but I feel like `newInstance` should instead take a single argument with the screen we want, e.g. `enum SettingsScreen { TOP_LEVEL, ... }`.

Reply:
I like that idea! But yeah, I don't want to do it in this fix so I filed #1671 for it.